### PR TITLE
Updated BetterCI Config

### PR DESCRIPTION
**Pull request by gdsoumya**
Signed-off-by: Soumya Ghosh Dastidar <gdsoumya@gmail.com>

## Proposed changes

Updated BetterCI config

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [x] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [x] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the commit for DCO to be passed.
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: .betterci/ci-k8s-manifest.yml [hunk 1]
```diff
@@ -17,10 +27,10 @@ spec:
     spec:
       containers:
         - name: litmusportal-frontend
-          image: #{portal-frontend}
+          image: litmuschaos/litmusportal-frontend:ci
```
Comment (rajdas98) [inline .betterci/ci-k8s-manifest.yml:30]: Why hardcoding of images?

**Review by rajdas98 — CHANGES_REQUESTED** [review]

File: .betterci/ci-k8s-manifest.yml (hunk 1 above)
Comment (gdsoumya) [reply .betterci/ci-k8s-manifest.yml]: Fixed it

**Review by rajdas98 — APPROVED** [review]

File: .betterci/ci-k8s-manifest.yml [hunk 2]
```diff
@@ -56,33 +66,63 @@ spec:
...
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+            - name: PORTAL_SCOPE
+              value: "cluster"
             - name: SUBSCRIBER_IMAGE
-              value: #{subscriber}
-            - name: SUBSCRIBER_NAMESPACE
-              value: "litmus"
+              value: "litmuschaos/litmusportal-subscriber:ci"
```
Comment (rajdas98) [inline .betterci/ci-k8s-manifest.yml:97]: SUBSCRIBER_IMAGE and DEPLOYER_IMAGE should also be a template env. Right?

**Review by rajdas98 — CHANGES_REQUESTED** [review]

File: .betterci/ci-k8s-manifest.yml (hunk 2 above)
Comment (gdsoumya) [reply .betterci/ci-k8s-manifest.yml:97]: Updated subscriber image, self_deployer image is not required as self_cluster isn't isn't set to true here.

**Review by rajdas98 — APPROVED** [review]

**Pull request merged**